app/controllers/api/v1/invoices/item_invoice_controller.rb

```diff
@@ -0,0 +1,8 @@
+class Api::V1::Invoices::ItemInvoiceController < ApplicationController
```

Comment:
@Dpalazzari Since this is namespaced under invoices. can we just call it the items controller?